tools/tracing/performance/README.md

```diff
@@ -152,15 +152,16 @@ If you can't tell which lock instance is for which lock in MMTk, you can trace
 the allocation of the Mutex and record the stack trace (note that you might want
 to compile MMTk with `force-frame-pointers` to obtain better stack traces).
 
-### Measuring the distribution of `process_edges` packet sizes (`packet_size`)
-Most of the GC time is spend in the transitive closure for tracing-based GCs,
-and MMTk performs transitive closure via work packets that calls the `process_edges` method.
-This tool measures the distribution of the sizes of these work packets, and also
-count root edges separately.
+### Measuring the distribution of `process_slots` packet sizes (`packet_size`)
```

Comment:
That's what the work probe currently measures (i.e. the number of slots the work packet processes).  There should be a way to measure the number of *edges* it (and other methods) processes in a more general way (e.g. by counting the number of times `trace_object` is called), but I don't want to do that kind of refactoring in this PR.

Reply:
To my understanding, it measures the packet sizes for `ProcessEdgesWork` packets. The old title was inaccurate.

Reply:
> To my understanding, it measures the packet sizes for `ProcessEdgesWork` packets. The old title was inaccurate.

That's right.  The work packet is `ProcessEdgesWork`, and the method is `process_slots`.  I'll change the title to use the packet name instead of the method name.